org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/outline/ModelBuilder.scala

```diff
@@ -153,10 +153,15 @@ object ModelBuilder extends HasLogger {
       }
 
       def showName(name: Name, t: Tree) = {
-        if (src.content(t.pos.point) == '`')
-          "`" + name.decoded + "`"
-        else
-          name.decoded.split("\\.").reverse.head
+        t match {
+          case t: DefDef if t.name.toString == "<init>" =>
```

Comment:
nitpick: you can compare `t.name` with `nme.CONSTRUCTOR`. It's two Int comparisons, and you save both an allocation and a linear comparison via Strings :)